test/runtests-MultifilesLogger.jl

```diff
@@ -0,0 +1,99 @@
+using Pkg
+Pkg.activate(".")
+
+using Revise
```

Comment:
What is Revise used for here?

Reply:
So that we don't need to restart a julia session before running the content of  `test/runtests-MultifilesLogger.jl`.

This is indeed just for convenience for the developer working on IOLogging and the tests actually don't need it. I don't think I have added it to the dependencies of the project.

Reply:
When running tests with `]test` from the REPL, a new temporary environment is created in which the tests are run, so I don't think Revise is needed here. For example, if you activate the project, run `]test`, add a failing test with `@test false` to runtests.jl and run `]test` in the same REPL session, the new test will fail the test.

Reply:
ok, I remove it